server/utils/socket.js

```diff
@@ -8,9 +8,11 @@ module.exports = async (server, app) => {
   app.use(async (req, res, next) => {
     token = req.cookies.token;
 
-    const decoded = jwt.verify(token, process.env.JWT_SECRET);
+    if (token) {
+      const decoded = jwt.verify(token, process.env.JWT_SECRET);
 
-    currentUser = await User.findById(decoded.id);
+      currentUser = await User.findById(decoded.id);
+    }
 
     next();
```

Comment:
This is a problem, it will allow anyone access the route cause if they don't have the token then JavaScript will just not go into that conditional and hit the `next()` statement. We should just let jwt.verify fail or wrap this in a try/catch and reject the request in the catch condition

Reply:
Whoops - this isn't the protect middleware. Let's just remove this change from here since it's unrelated 🙂 🙏🏻

Reply:
Can we remove this file from the files changes since it's not related to the PR